lib/ex_aws/sns/parsers.ex

```diff
@@ -108,6 +139,39 @@ if Code.ensure_loaded?(SweetXml) do
                           subscription_arn: ~x"./SubscriptionArn/text()"s,
                           topic_arn: ~x"./TopicArn/text()"s,
                         ],
+                        next_token: ~x"./ListSubscriptionsResult/NextToken/text()"s,
```

Comment:
@jnicklas I added this in because it looks like it was missing. I added a separate test for it as well.